tests/core/test_serialization.py

```diff
@@ -696,6 +704,133 @@ def test_checkpoint_config_and_nested_objects_are_serialized(
     )
 
 
+@pytest.mark.unit
+def test_checkpoint_config_and_nested_objects_are_serialized_spark(spark_session):
```

Comment:
By nature of having a Spark session here, I think this is an `integration` test

Reply:
you're right. Updated